tests/stave_tests.ts

```diff
@@ -747,10 +747,10 @@ function configureAllLines(options: TestOptions, contextBuilder: ContextBuilder)
 function drawStaveText(options: TestOptions, contextBuilder: ContextBuilder): void {
   const ctx = contextBuilder(options.elementId, 900, 140);
   const stave = new Stave(300, 10, 300);
-  stave.setText('Violin', Modifier.Position.LEFT);
-  stave.setText('Right Text', Modifier.Position.RIGHT);
-  stave.setText('Above Text', Modifier.Position.ABOVE);
-  stave.setText('Below Text', Modifier.Position.BELOW);
+  stave.setStaveText('Violin', Modifier.Position.LEFT);
+  stave.setStaveText('Right Text', Modifier.Position.RIGHT);
+  stave.setStaveText('Above Text', Modifier.Position.ABOVE);
+  stave.setStaveText('Below Text', Modifier.Position.BELOW);
```

Comment:
Can you explain why it is now renamed to setStaveText?

It seems a bit redundant to say `stave.setStaveText(...)`. Why not just `stave.setText(...)`?

Reply:
Already answered :)